docs/pages/unified/marketing-automation/marketing-automation.yaml

```diff
@@ -351,19 +359,38 @@ paths:
             schema:
               type: object
               properties:
+                include:
+                  $ref: "#/components/schemas/IncludeFields"
                 id:
                   type: string
               required:
                 - id
             examples:
               example-1:
                 value:
-                  id: '123123'
+                  id: "123123"
       operationId: get-one-list
       description: Find List
     parameters: []
 components:
   schemas:
+    IncludeFields:
+      title: Include Fields
+      type: object
+      examples:
+        - $native: true
+        - {}
```

Comment:
what's the extra object for?

Reply:
I was using that to show you could leave it empty, but I think that's just more confusing, so I'll just get rid of it